Example/Podfile

```diff
@@ -11,3 +11,16 @@ target 'Colander_Example' do
     pod 'Nimble', '~> 8.0.1'
   end
 end
+
+post_install do |installer|
```

Comment:
Nimble had a linker issue which works around. This is fixed in later versions, so this should be updated as a follow up, but I didn't want to add even more to clutter this PR.